### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Laguna1**
We have completed this project and we will be happy to receive feedback from you, we have checked the HTML markup, and our CSS styles with a validator, please let us know if we have anything else to fix, we will wait for your prompt reply.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,774 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="assets/reset.css">
```
Comment (pelzolga123) [inline index.html]: You don't have `reset.css` file in your project, so it's better to delete link to it.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: great!
Here are things you need to change before this project can be approved:
* [ ] Pull Request should have meaningful title and a short description (screenshot from Pathwright) Right now your title is not really meaningful, and description should have list of changes you implemented while working on this project.
* [x] The NY Times logo should be a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))
* [x] All elements should be aligned properly:
* `six-1` div:
image

 Original:
image

* video inside `eight-1` div:
image

 Original:
image

* `tenth` div:
image

Original:
image

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by Laguna1** [comment]
Thanks Olga!
Your comments as always help a lot,
Of course we will fix it.

пн, 27 янв. 2020 г. в 19:52, Olga Pelts <notifications@github.com>:

**Pull request merged**